product_subscription_mass_mailing/models/mailing_criterium.py

```diff
@@ -69,9 +69,12 @@ class MailingCriterium(models.Model):
     )
 
     def get_subscriptions(self):
-        subscriptions = self.env["product.subscription.object"].search(
-            [("template", "=", self.template.id)]
-        )
+        if self.template:
+            domain = [("template", "=", self.template.id)]
+        else:
+            domain = [("is_trial", "!=", False)]
+
+        subscriptions = self.env["product.subscription.object"].search(domain)
```

Comment:
do we need an ensure one here ?

Reply:
no because it's not called via @api.multi: it's called from a button or from `def send_email` which loops on self.